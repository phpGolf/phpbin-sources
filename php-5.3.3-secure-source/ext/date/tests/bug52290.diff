002+ string(47) "2041-W52-4 | 2041-12-26 | 00:00:00 | 2271628800"
003+ string(47) "2044-W40-1 | 2044-10-03 | 00:00:00 | 2359065600"
004+ string(47) "2081-W39-6 | 2081-09-27 | 20:30:40 | 3526230640"
002- string(47) "2005-W52-1 | 2005-12-26 | 00:00:00 | 1135555200"
003- string(47) "2007-W40-5 | 2007-10-10 | 00:00:00 | 1191974400"
004- string(47) "2007-W40-5 | 2007-10-10 | 20:30:40 | 1192048240"