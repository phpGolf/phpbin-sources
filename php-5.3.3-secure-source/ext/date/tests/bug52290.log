
---- EXPECTED OUTPUT
string(47) "2005-W52-7 | 2006-01-01 | 00:00:00 | 1136073600"
string(47) "2005-W52-1 | 2005-12-26 | 00:00:00 | 1135555200"
string(47) "2007-W40-5 | 2007-10-10 | 00:00:00 | 1191974400"
string(47) "2007-W40-5 | 2007-10-10 | 20:30:40 | 1192048240"
---- ACTUAL OUTPUT
string(47) "2005-W52-7 | 2006-01-01 | 00:00:00 | 1136073600"
string(47) "2041-W52-4 | 2041-12-26 | 00:00:00 | 2271628800"
string(47) "2044-W40-1 | 2044-10-03 | 00:00:00 | 2359065600"
string(47) "2081-W39-6 | 2081-09-27 | 20:30:40 | 3526230640"
---- FAILED
