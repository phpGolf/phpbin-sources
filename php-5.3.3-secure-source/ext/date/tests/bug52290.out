string(47) "2005-W52-7 | 2006-01-01 | 00:00:00 | 1136073600"
string(47) "2041-W52-4 | 2041-12-26 | 00:00:00 | 2271628800"
string(47) "2044-W40-1 | 2044-10-03 | 00:00:00 | 2359065600"
string(47) "2081-W39-6 | 2081-09-27 | 20:30:40 | 3526230640"